webapp/components/admin_console/legal_and_support_settings.jsx

```diff
@@ -79,7 +79,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.privacyDesc'
-                            defaultMessage='Link to Privacy Policy available to users on desktop and on mobile. Leaving this blank will hide the option to display a notice.'
+                            defaultMessage='Link to Privacy Policy available to users on desktop and on mobile. Leaving this field blank will hide the option to display a notice.'
```

Comment:
The URL for the Privacy link on the login and sign-up pages. If this field is empty, the Privacy link is hidden from users.